### PR TITLE
Bug 1175461 - no longer setting dialog height manually

### PR DESCRIPTION
**Pull request by jedireza**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fab2c570de4c607dfcc783aa42f0db584fe94fd2

File: apps/system/js/modal_dialog.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ var ModalDialog = {
 
     this.screen = document.getElementById('screen');
     this.overlay = document.getElementById('dialog-overlay');
+    this.overlay.style.height = 'auto';
```
Comment (mikehenrty) [inline apps/system/js/modal_dialog.js]: Why not do this in css?

Comment (jedireza) [inline apps/system/js/modal_dialog.js]: We sure could. Firstly I wanted to validate that the approach was acceptable.

**Comment by jedireza** [comment]
Closing (for now) in favor of #30993

**Pull request closed without merging**